### PR TITLE
Add Coveralls & Stylish Haskell to CI

### PR DESCRIPTION
**Pull request by KtorZ**
# Issue Number

#1

# Overview

- [ ] I have added checks for stylish-haskell in the CI
- [ ] I have added support for coveralls in the CI
- [ ] I have removed redundant dependencies from the .cabal file (make hlint and weeder happy)
- [ ] I have disabled build on all branches but master to avoid starting a new job every time someone pushes something on a branch.
- [ ] I have added the coveralls badge to the README back again (because badges are of primary importance) 

# Comments

### CODE REVIEW

**Comment by Anviking** [comment]
https://coveralls.io/github/input-output-hk/cardano-wallet

Coveralls seems a bit confused, probably because of the archived `cardano-wallet`.

skarmavbild 2019-03-04 kl 14 14 12

**Comment by Anviking** [comment]
Maybe it goes away after merging…?

File: cardano-wallet.cabal [hunk 1]
```diff
@@ -28,28 +28,6 @@ library
   build-depends:
       base
 
-    -- Hackage Dependencies
-    , aeson
```
Comment (Anviking) [inline cardano-wallet.cabal:32]: Sure 👍

**Review by Anviking — APPROVED** [review]

**Comment by KtorZ** [comment]
I believe Coveralls will sync up over time but yeah, it got a bit confused in the battle.

**Pull request merged**